index.html

```diff
@@ -113,11 +113,11 @@
     //-------------------------------------------------------------------------
 
     var i = { size: 4, blocks: [0x0F00, 0x2222, 0x00F0, 0x4444], color: 'red'   };
-    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'yellow'   };
-    var l = { size: 3, blocks: [0x4460, 0x0E80, 0xC440, 0x2E00], color: 'green' };
-    var o = { size: 2, blocks: [0xCC00, 0xCC00, 0xCC00, 0xCC00], color: 'orange' };
+    var j = { size: 3, blocks: [0x44C0, 0x8E00, 0x6440, 0x0E20], color: 'red'   };
```

Comment:
So, we don't want the game to use multiple pieces that use the same colors (colours). If you change the colors to a bunch of different colors, it will work workworkworkworkwork